src/RealtimeServer/common/models/project-rights.ts

```diff
@@ -12,10 +12,7 @@ export enum Operation {
   ViewOwn = 'view_own'
 }
 
-export interface ProjectRight {
-  projectDomain: string;
-  operation: Operation;
-}
+export type ProjectRight = [string, `${Operation}`];
```

Comment:
Should we label the tuple elements? i.e. 
```typescript
export type ProjectRight = [projectDomain: string, operation: `${Operation}`];
```
See https://devblogs.microsoft.com/typescript/announcing-typescript-4-0/#labeled-tuple-elements